src/Components/Auth/Login.tsx

```diff
@@ -282,7 +282,7 @@ export const Login = (props: { forgot?: boolean }) => {
                   forgotPassword && "hidden",
                 )}
               >
-                <div className="mb-8 w-[300px] text-4xl font-black text-primary-600">
+                <div className="mb-8 w-[250px] max-w-full text-[min(10vw,1.85rem)] font-black text-primary-600 lg:w-[300px] lg:text-4xl">
```

Comment:
Prefer `rem` values over hardcoded pixel values. Pixel values may produce inconsistent results on different browser/screen zoom settings.

Reply:
@rithviknishad Switched to `w-64` (256px) &  `w-72` (288px)